study-builder/studies/testboston/i18n/en.conf

```diff
@@ -3,170 +3,73 @@
     "error": "We're sorry. That zip code is outside the TestBoston study zone. Please check the zip code you entered and try again.",
     "warning": "The address you entered is outside the current delivery area for TestBoston. If you believe there has been a mistake, please contact TestBoston study staff: info@testboston.org or 617-525-8418."
   },
-
   "settings": {
     "invite_error": "The code you entered isn't active. Please check your code and try again. You can always contact the TestBoston study staff for help: info@testboston.org or 617-525-8418."
-  }
-
+  },
   "consent": {
     "name": "Consent Form",
-    "title": """Consent Process<a class="button button_download" href="consent.pdf" target="_blank"><img src="assets/images/download.svg" alt="Download" class="activity-header__icon">Download information sheet</a>""",
+    "title": "Consent Process<a class=\"button button_download\" href=\"consent.pdf\" target=\"_blank\"><img src=\"assets/images/download.svg\" alt=\"Download\" class=\"activity-header__icon\">Download information sheet</a>",
     "subtitle": "",
     "last_updated": "TestBoston Version 1.0",
-    "readonly_hint": """
-      Thank you for signing your consent form. If you would like to make any changes,
-      please reach out to the study team.""",
-
-    // Section 1
-
```

Comment:
I normalized the whitespace and quotes here to make it easier to compare against other i18n files.